tests/test_filtering.py

```diff
@@ -0,0 +1,87 @@
+from kbmod import *
+import numpy as np
+import tempfile
+import unittest
+
+class test_kernels_wrappers(unittest.TestCase):        
+    def test_sigmag_filtered_indices_same(self):
+        # With everything the same, nothing should be filtered.
+        values = [1.0 for _ in range(20)]
+        inds = sigmag_filtered_indices(values, 0.25, 0.75, 0.7413);
+        self.assertEqual(len(inds), 20)
+        for idx in inds:
```

Comment:
Is this test supposed to be here? I thought everything in the loop should just be equal to 1.0?

Reply:
I'm not sure why that test is there anymore. I will remove it.